data/pug/pug.txt

```diff
@@ -609,7 +609,7 @@ ship "Pug Enfolta"
 	explode "pug medium explosion" 25
 	explode "pug large explosion" 15
 	"final explode" "final explosion medium"
-	description "This is a medium-sized Pug warship."
+	description `The Enfolta is a medium-sized Pug warship. It has a powerful arnament of advanced weaponry supported by powerful integrated energy generation. A single Enfolta can take on a fleet of smaller human ships.`
```

Comment:
```suggestion
	description `The Enfolta is a medium-sized Pug warship. It has a potent armament of advanced weaponry supported by powerful integrated energy generation. A single Enfolta can take on a fleet of smaller human ships.`
```

Reply:
I think there might be a better word than potent or powerful

Reply:
@roadrunner56 do you think "formidable" would be a good word?

Reply:
Yeah, that would be better.

Reply:
> 

Alright, I will change it to that.